minethd.cpp

```diff
@@ -33,7 +33,11 @@
 
 void thd_setaffinity(std::thread::native_handle_type h, uint64_t cpu_id)
 {
+#ifdef __MINGW64__
+	SetThreadAffinityMask(&h, 1ULL << cpu_id);
```

Comment:
I can't verify it because I have no MinGW, I have only cleaned up this PR for the user.